### ISSUE TITLE
[MAS4.2.12][Screen reader - Save trancript] No confirmation is provided when the transcript is saved.

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#2053]
Actual Result:
Voice over does not announce the confirmation after activating the save transcript control.

Expected Result:
Proper   confirmation should be given by VO when the transcript has been saved.

My Bots' section.
Test Environment
Platform: Mac 10.15.3 Beta (19D49f)
App: Bot Framework Emulator V4
App version: Version 4.7.0-99577 (4.7.0-99577.99577)
Tool: Voice Over

Repro Steps:
1.Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Create a new bot Using shortcut key using Ctr+N
4. Navigate on save transcript button and activate this button
5. Observe the voice over announcement.
 
User Impact:
Userwill unable to know once that transcript has been saved or not.

Impacted MAS Link:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b19c05bd-d82f-4d6d-a8e5-7081bf5c2dba}

Attachment
[confrimation announcement- DT issue.zip](https://github.com/microsoft/BotFramework-Emulator/files/4070755/confrimation.announcement-.DT.issue.zip)

**Comment by v-rampul** [comment]
#HCL-BotFrameworkEmulator; #Emulator-MAC; #A11yMAS; #HCL; #MAS4.2.12; #A11ySev4
